### ISSUE TITLE
Similar names for work item type groups and work item types could confuse end users

### ISSUE THREAD

**Issue by Preeticp** [openshiftio/openshift.io#3170]
In the left pane of Planner the work item type groups, Scenarios and Experiences, should have different names than the work item types they group together. Scenarios club together Scenario, Fundamental and Papercuts. Experiences club together Experience and Value-Proposition.

![screenshot from 2018-04-17 16-14-52](https://user-images.githubusercontent.com/16647625/38871045-437ec534-426d-11e8-962f-de11b51fca4f.png)

We should have something similar to Requirements which clubs together Feature, Bug and Task.

The confusion will further increase once Info-tips come into the picture. Seeing a different Info-tip for Scenario in the right pane and Scenarios in the left pane could be potentially confusing for the end user.

**Comment by rkratky** [comment]
@debloper Not sure about the reasoning here. This issue reacts to the way it is implemented in prod at this moment, which, IMHO, makes it a bug.

Ad the issue itself: I've run this past a number of people who are not familiar with OSIO. I only told them the following: _"We have scenarios, fundamentals, papercuts, experiences, and value props. Show me where you would look for each of those."_ The answer was always:

_"Here are the **Scenarios**, and here are the **Experiences**. I don't know where there rest is, but I see there are some **Requirements**, so I guess I would go there next."_

In other words, I think this is unintuitive and confusing.

**Comment by debloper** [comment]
@rkratky please don't shoot the messenger - I personally think the group-names on the left-pane should change as well to capture as the umbrella for all that it contains.

My point being, whether the names would/should be changed or what they'd become will need a planning first. Calling it bug means it doesn't behave the way it's meant to. But according to the last decision made (with UX approval) happens to be the exact way they are implemented right now. So, one can't call it a bug. You can call it fault/shortsightedness in previous planning process, but it's not an unintended behavior by software.

**Comment by rkratky** [comment]
@debloper Understood, thanks for the clarification.

**Comment by joshuawilson** [comment]
@debloper `type/planning` is for an issue that does planning. An issue that is a list of tasks to be done by a team. This issue could be a `type/feature-request` or maybe a `type/question`. But it is definitely NOT `type/planning`.

**Comment by nimishamukherjee** [comment]
Closing as this is related to SDD and we are not supporting that template.
